extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Anchor.fm integration.
+ *
+ * @since 9.0.0
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\AnchorFm;
+
+use Automattic\Jetpack\Blocks;
+use Jetpack_Podcast_Helper;
+
+const FEATURE_NAME = 'anchor-fm';
+const BLOCK_NAME   = 'jetpack/' . FEATURE_NAME;
+
+/**
+ * Registers Anchor.fm integration for the block editor.
+ */
+function register_block() {
+	Blocks::jetpack_register_block(
+		BLOCK_NAME,
+		array(
+			'plan_check' => false,
+		)
+	);
+
+	// Register post_meta for connecting Anchor podcasts with posts.
+	register_post_meta(
+		'post',
+		'anchor_podcast',
```

Comment:
I wonder if those should be prefixed with `jetpack_`, in case other plugins come along with similar features?